core/commands/root.go

```diff
@@ -61,13 +61,15 @@ TOOL COMMANDS
 Use 'ipfs <command> --help' to learn more about each command.
 
 ipfs uses a repository in the local file system. By default, the repo is located
-at ~/.ipfs. To change the repo location, set the $IPFS_PATH environment variable:
+at ~/.ipfs. To change the repo location, you can either use --repo or set the $IPFS_PATH environment variable:
+
+    ipfs <cmd> --repo=/path/to/ipfsrepo
 
   export IPFS_PATH=/path/to/ipfsrepo
 `,
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("config", "c", "Path to the configuration file to use."),
+		cmds.StringOption("repo", "R", "Path to the configuration file to use."),
```

Comment:
`--repo` is the better name though, because the option is supposed to point to the repo, not the config file :)